p2d/p2d_utils.py

```diff
@@ -21,6 +21,48 @@
 RESOURCES_PATH = os.path.join(
     os.path.split(os.path.realpath(__file__))[0], 'resources')
 
+# Custom formatter for the console logger handler.
+# Adapted from https://stackoverflow.com/questions/384076/how-can-i-color-python-logging-output
+class Pol2DomLoggingFormatter(logging.Formatter):
+    INDENT = 0
+
+    bold_gray = '\033[1;90m'
+    bold_green = '\033[1;92m'
+    bold_yellow = '\033[1;33m'
+    bold_red = '\033[1;91m'
+    yellow = '\033[33m'
+    red = '\033[91m'
+    reset = '\033[0m'
+
+    COLORS = {
+        logging.DEBUG: (bold_gray, reset),
+        logging.INFO: (bold_green, reset),
+        logging.WARNING: (bold_yellow, yellow),
+        logging.ERROR: (bold_red, red)
+    }
+
+    def format(self, record):
+        level_color, message_color = self.COLORS.get(record.levelno)
+        padding = ' ' * (10 - len(record.levelname))
+        log_fmt = '  ' * self.INDENT + level_color + '{levelname}' + self.reset + padding + message_color + '{message}' + self.reset
+        formatter = logging.Formatter(log_fmt, style='{')
+        return formatter.format(record)
+
+def configure_logging(verbosity):
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel(eval('logging.' + verbosity.upper()))
+    console_handler.setFormatter(Pol2DomLoggingFormatter())
+
+    logging.basicConfig(
+        format='{levelname}\t{message}',
+        style='{',
+        level=eval('logging.' + verbosity.upper()),
+        handlers=[console_handler]
```

Comment:
Found out you can just do this and keep using logging everywhere, instead of instantiating an object of type logging.Logger, which is more cumbersome.